tools/slang-test/main.cpp

```diff
@@ -800,7 +509,7 @@ TestResult runSimpleTest(TestContext* context, TestInput& input)
     // Otherwise we compare to the expected output
     if (actualOutput != expectedOutput)
     {
-        context->dumpOutputDifference(expectedOutput, actualOutput);
+        context->reporter->dumpOutputDifference(expectedOutput, actualOutput);
```

Comment:
For stuff like this, providing a forwarding method to minimize the diffs seems like it would be a nice simplification.